llvm/include/llvm/CodeGen/LiveIntervals.h

```diff
@@ -113,14 +114,18 @@ class LiveIntervals {
   ~LiveIntervals();
 
   /// Calculate the spill weight to assign to a single instruction.
+  /// If \p PSI is provided the calculation is altered for optsize functions.
```

Comment:
I understand that this may be confusing, but I thought it was less disruptive than adding a separate `ConsiderOptSize` parameter